Create jpackage_config.txt

gonzonia opened the pull request:
Created config to pass to jpackage. Doesn't work. Creates app bundle but does not use Zulu so app doesn't launch.

gonzonia committed:
Create jpackage_config.txt

Created config to pass to jpackage. Doesn't work. Creates app bundle but does not use Zulu so app doesn't launch.

gonzonia committed:
JRE added

Updated to include JRE correctly.

gonzonia committed:
Adding icon files for packaging

These are some files I used in testing. They might need to be replaced.

gonzonia committed:
Create VPin-Studio.zip

App bundle created with jpackage-config and icon files.

gonzonia committed:
Merge branch 'main' into mac-package

gonzonia committed:
Replacing corrupt files

Files didn't upload correctly. Trying again.

gonzonia committed:
Merge branch 'mac-package' of https://github.com/gonzonia/vpin-studio into mac-package

gonzonia commented:
Moved zip of app bundle to Google Drive. 
https://drive.google.com/drive/folders/1S4FvJSF3psJ0YZWkhS-oTTZxp3JtLDnW?usp=share_link

karlsnyder0 commented:
@gonzonia  The app bundle template will look like this (attached zip). We should be able to commit this file into the repository and use it as a baseline for all macOS App builds.

During the build we should create a VPin-Studio-Client-macosx_aarch64_app.zip for the macOS runner to pick up.

/Contents/app - Vpin Studio app files go here (example /Contents/app/vpin-studio-ui.jar)
/Contents/runtime - Zulu jre zip Contents go here (example /Contents/runtime/Contents)

[VPin-Studio-Client-macosx_aarch64_app_bundle_template..zip](https://github.com/user-attachments/files/18311863/VPin-Studio-Client-macosx_aarch64_app_bundle_template.zip)

gonzonia committed:
Merge branch 'main' into mac-package

gonzonia committed:
Updating with template

Template of Mac App bundle.

gonzonia committed:
Update pom.xml

gonzonia committed:
Update of Jpackage file and draft of workflow

I made a tweak to the jpackage file to hopefully work with the mac runner and the workflow.yml. I know some of the relative paths will need to be adjusted in both the workflow and jpackage files

gonzonia committed:
Create macos-create-app-and-codesign.yml

This is a first draft of a workflow for creating a Mac app bundle. Definitely needs adjustment.

gonzonia committed:
Delete workflow.yml

gonzonia committed:
Merge branch 'main' into mac-package

gonzonia committed:
Merge branch 'mac-package' of https://github.com/gonzonia/vpin-studio into mac-package
